### PR TITLE
Build PDF Book

### PR DESCRIPTION
**Pull request by sudo-rsingh**

### CODE REVIEW

File: new-website/utils/tutorials/build_pdf_book.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+"""
```
Comment (rbharath) [inline new-website/utils/tutorials/build_pdf_book.py:1]: @GreatRSingh Add a section to the documentation / README on the PDF building capability

File: new-website/utils/tutorials/build_pdf_book.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+This script is used to build the pdf book from DeepChem Tutorials.
+
+Requirements:
+    - pdfunite
```
Comment (rbharath) [inline new-website/utils/tutorials/build_pdf_book.py:5]: Add this to our requirements.txt

**Review by rbharath — COMMENTED** [review]
@GreatRSingh A few comments but this is looking mostly good

File: new-website/utils/tutorials/build_pdf_book.py (hunk 2 above)
Comment (sudo-rsingh) [reply new-website/utils/tutorials/build_pdf_book.py:5]: pdfunite is a linux application which i am using to merge pdfs.
I added pdfkit to requirements.txt.

**Review by rbharath — APPROVED** [review]
LGTM

**Pull request merged**